dev-java/spec-alpha/spec-alpha-0.2.194.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit java-pkg-2 java-ant-2
+
+EGIT_REF="fb55378"
+
+MY_P=spec.alpha-${PV}
+
+DESCRIPTION="A Clojure library to describe the structure of data and functions."
+HOMEPAGE="https://clojure.org/ https://github.com/clojure/spec.alpha"
+SRC_URI="https://github.com/clojure/spec.alpha/archive/${MY_P}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="EPL-1.0"
+SLOT="0.2"
+KEYWORDS="~amd64 ~x86 ~x86-linux"
+IUSE=""
+
+COMMON_DEPEND="dev-java/ant-core:0"
+RDEPEND=">=virtual/jre-1.8:*"
+DEPEND="
+	${COMMON_DEPEND}
+	>=virtual/jdk-1.8:*"
```

Comment:
It doesn't look like `COMMON_DEPEND` is needed here at all.

Reply:
Moved dev-java/ant-core:0 to DEPEND